pkgs/tools/inputmethods/keyd/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv, python39Packages, python39, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "keyd";
+  version = "2.4.0";
+
+  src = fetchFromGitHub {
+    owner = "rvaiya";
+    repo = "keyd";
+    rev = "v${version}";
+    sha256 = "sha256-YZSwXDrzly5OlNh8j+Y2/qYmGHXE5T3ZAvq7e45jVL0=";
+  };
+
+  makeFlags = [ "DESTDIR=${placeholder "out"}" "PREFIX=" ];
```

Comment:
```suggestion
  makeFlags = [ "PREFIX=${placeholder "out"}" ];
```

Reply:
I only took a quick look at the Makefile. If we can remove PREFIX and it works I think we are good here.